espnet2/asr_transducer/espnet_transducer_model.py

```diff
@@ -362,39 +419,144 @@ def _calc_transducer_loss(
                 )
             except ImportError:
                 logging.error(
-                    "warp-rnnt was not installed."
+                    "warp-transducer was not installed. "
                     "Please consult the installation documentation."
                 )
                 exit(1)
 
-        loss_transducer = self.criterion_transducer(
-            joint_out,
-            target,
-            t_len,
-            u_len,
-        )
+        with autocast(False):
```

Comment:
It's an oversee on my part, it should have been included in #5140 for amp usage. The (auto)castings will be changed when float16 support is merged to `warp-transducer`.